website/docs/components/modal/partials/guidelines/guidelines.md

```diff
@@ -172,23 +172,23 @@ The purpose and function of the Modal should not rely solely on an icon, instead
 
 An icon paired with the title can help reinforce the purpose and function of the Modal while also drawing the eye to the header and title area. Icons can be used to communicate the severity and importance of interacting with a Modal and are especially useful in a **warning** or **critical** color Modal.
 
-<Hds::Modal::Header @icon="info" @onDismiss={{this.noop}}>Title</Hds::Modal::Header>
+![Modal header title with icon and dismiss button](/assets/components/dialog-primitives/dialog-primitives-header-icon-and-title.jpg)
 
 #### Without title icon
 
-<Hds::Modal::Header @onDismiss={{this.noop}}>Title</Hds::Modal::Header>
+![Modal header title only and dismiss button](/assets/components/dialog-primitives/dialog-primitives-header-title-only.jpg)
```

Comment:
```suggestion
![Modal header title with only dismiss button](/assets/components/dialog-primitives/dialog-primitives-header-title-only.jpg)
```

Reply:
Leaving the original version, so resolving this now.